pkg/api/v1/users.go

```diff
@@ -48,7 +50,10 @@ func NewUser(model *models.User) (out *User, err error) {
 
 	var roles []*models.Role
 	if roles, err = model.Roles(); err != nil {
-		return nil, err
+		if !errors.Is(err, qde.ErrMissingAssociation) {
```

Comment:
I pretty much copied this: https://github.com/rotationalio/quarterdeck/blob/1971dc010c867d459631de2ad1c1fd9481fdcb88/pkg/store/sqlite/users.go#L106